x/oracle/keeper/hooks.go

```diff
@@ -44,3 +48,29 @@ func (h Hooks) AfterMarketGenesis(ctx sdk.Context, tickers map[string]marketmapt
 
 	return nil
 }
+
+// AfterMarketCreated is the marketmap hook for x/oracle that is run after a market is created in
+// the marketmap.  After the market is created, a currency pair and its state are initialized in the
+// oracle module.
+func (h Hooks) AfterMarketCreated(ctx sdk.Context, market mm2types.Market) error {
```

Comment:
Probably want a bunch of e2e tests in the future covering parity between mm and x/oracle.

Reply:
our e2e test suite covers this once we do the switch from `mm2` to `marketmap`